tests/python/gaia-ui-tests/gaiatest/apps/base.py

```diff
@@ -103,24 +104,37 @@ def a11y_select(self, match_string):
         self.accessibility.click(self.marionette.find_element(*_close_button_locator))
         self.wait_for_select_closed(*_close_button_locator)
 
-    def tap_element_from_system_app(self, element=None, add_statusbar_height=False, x=None, y=None):        # Workaround for bug 1109213, where tapping on the button inside the app itself
+    def tap_element_from_system_app(self, element=None, add_statusbar_height=False, x=None, y=None):
```

Comment:
This shouldn't be happening, this pull request shouldn't contain any tap_element_from_system_app changes.